README.md

```diff
@@ -47,6 +47,9 @@ All messages                         | [Stable](https://github.com/open-telemetr
 (See [maturity-matrix.yaml](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57)
 for definition of maturity levels).
 
+Note: 1.0.0 and newer releases from this repository may contain unstable (alpha or beta)
+components as indicated by the Maturity table above.
+
```

Comment:
Suggestion: remove "Note:" and move this sentence right after `## Maturity Level`